README.md

```diff
@@ -4,7 +4,9 @@
 
 This library declares a `checks()` function and a `checkers` table, which
 allow to check the parameters passed to a Lua function in a fast and
-unobtrusive  way.
+unobtrusive way.
```

Comment:
And let's restore a separate line about "Starting with 2.11.0, Tarantool embeds the `checks` module."

Reply:
done